package/README.md

```diff
@@ -51,29 +51,39 @@ Here is an overview of the release process.
 
    * Not all commit descriptions which come from `git log` will explain changes well. Reword items as necessary, with the goal that a general user of this project should be able to understand what the change is related to. Preserve references to PRs or issues (`#XXXX`).
 
-   * Don't include fixes for issues which have been introduced since last release.
+   * Don't include fixes for regressions which have been introduced since last release.
 
    * Aggregate minor fixes (e.g. typos, small documentation changes) in a few items. Focus on preparing a good overview of the release for the users, rather than mentioning every change.
 
    * When done, put release notes into a private Gist and send the link to other maintainers for review.
 
-2. Tag the latest commit. In this project, tags have form `X.Y.Z`, e.g. `2.4.0`, or `X.Y.Z-rcN` for release versions. Notice that there's no `v`at the beginning of the tag. Tags must be annotated, not lightweight tags. To create a tag, use git command (assuming that the master branch is checked out):
+2. Create a commit on master branch (directly or via a Pull Request), making the following changes:
+
+   * Set version in platform.txt to the version about to be released. E.g. change from `2.4.0-dev` to `2.4.0`
+
+   * In main README.md, in "Latest release" section, change version number in the readthedocs link to the version which is about to be created.
+
+   * Commit message should be `Release X.Y.Z`
+
+3. Tag the aboce commit, once it is on the master branch. In this project, tags have form `X.Y.Z`, e.g. `2.4.0`, or `X.Y.Z-rcN` for release versions. Notice that there's no `v`at the beginning of the tag. Tags must be annotated, not lightweight tags. To create a tag, use git command (assuming that the master branch is checked out):
```

Comment:
Ideally it is just "create a commit, tag it, and push it onto the master branch", but if you wish to do this via a PR, it can be "create a commit and push it onto a new branch, open a PR, wait for it to be green for merging, merge PR (using "rebase" strategy), tag the newly created commit on the master branch". Should I explain both workflows or just the simple one (commit to master)?

Reply:
@igrr Just the simple one please.

Reply:
BTW, I _think_ I understand what needs to be done, but I can't help thinking of someone else reading this down the line.